Tests/BackendIntegrationTests/MainThreadMonitor.swift

```diff
@@ -50,3 +55,22 @@ final class MainThreadMonitor {
     private static let threshold: DispatchTimeInterval = .seconds(1)
 
 }
+
+private extension MainThreadMonitor {
+
+    // From https://stackoverflow.com/a/33177600/401024
+    static var debuggerIsAttached: Bool {
+        // Buffer for "sysctl(...)" call's result.
+        var info = kinfo_proc()
+        // Counts buffer's size in bytes (like C/C++'s `sizeof`).
+        var size = MemoryLayout.stride(ofValue: info)
+        // Tells we want info about own process.
+        var mib: [Int32] = [CTL_KERN, KERN_PROC, KERN_PROC_PID, getpid()]
```

Comment:
Not sure but since this info isn't supposed to change, could `mib` be a `let`?

Reply:
It's passed as a reference in the next line so it can be modified. Swift would actually suggest this needs to be immutable otherwise :)